src/modules/helpchan.ts

```diff
@@ -286,32 +285,20 @@ export class HelpChanModule extends Module {
 		);
 	}
 
-	@command()
-	async claim(msg: Message, user: GuildMember) {
-		if (!msg.guild || !msg.member || msg.channel.type !== 'text') {
-			return;
-		}
-
-		// We'll want to refactor this into an inhibitor someday.
-		if (
-			!msg.member.roles.cache.has(trustedRoleId) &&
-			!msg.member.hasPermission('MANAGE_MESSAGES')
-		) {
-			return msg.channel.send(
-				"You don't have permission to use that command.",
-			);
-		}
-
-		const helpUser = await HelpUser.findOne({ where: { userId: user.id } });
+	@command({ inhibitors: [isTrustedMember] })
+	async claim(msg: Message, member: GuildMember) {
+		const helpUser = await HelpUser.findOne({
+			userId: member.id,
+		});
 		if (helpUser) {
 			return msg.channel.send(
-				`${user.displayName} already has an open help channel: <#${helpUser.channelId}>`,
+				`${member.displayName} already has an open help channel: <#${helpUser.channelId}>`,
 			);
 		}
 
 		const channelMessages = await msg.channel.messages.fetch({ limit: 50 });
 		const questionMessages = channelMessages.filter(
-			msg => msg.author.id === user.id,
+			({ author, id }) => author.id === member.id && id !== msg.id,
```

Comment:
This is quite confusing to read IMO, maybe no deconstruction and just call it something that doesn't conflict with the existing `msg`?